libs/services/Converter.py

```diff
@@ -6,6 +9,7 @@ class Converter:
     def __init__(self, session):
         self.session = session
 
+    @lru_cache
```

Comment:
Are you sure `self` is hashable here? It contains the session.

Reply:
You are right, we have to make sure default hash based on `id` is sufficient here. We will have a look into that in #1 (cache testing).